lms/djangoapps/django_comment_client/permissions.py

```diff
@@ -13,6 +14,12 @@
 from teams.models import CourseTeam
 
 
+class ThreadContext(Enum):
```

Comment:
Should we put this in common/djangoapps/django_comment_common so it is accessible in common if necessary? I don't really understand having it in the permissions file.